src/components/rectification/RectificationForm.tsx

```diff
@@ -23,11 +28,16 @@ const RectificationForm = () => {
 
   const { getUser } = useClient();
   const user = getUser();
+  const selectedForm = useSelector(selectFormContent).selectedForm;
+  const movedCarFormSelected = selectedForm == FormId.MOVEDCAR;
```

Comment:
```suggestion
  const movedCarFormSelected = selectedForm === FormId.MOVEDCAR;
```

This could be a strict equal for type safety